apps/client/src/common/stores/appModeStore.ts

```diff
@@ -9,7 +9,18 @@ export enum AppMode {
 const appModeKey = 'ontime-app-mode';
 
 function getModeFromSession() {
-  return sessionStorage.getItem(appModeKey) === AppMode.Run ? AppMode.Run : AppMode.Edit;
+  const appModeFromSessionStorage = sessionStorage.getItem(appModeKey);
+
+  switch (appModeFromSessionStorage) {
+    case AppMode.Run:
+      return AppMode.Run;
+    case AppMode.Edit:
+      return AppMode.Edit;
+    case AppMode.Freeze:
+      return AppMode.Freeze;
+    default:
+      return AppMode.Run;
+  }
```

Comment:
in the domain, run and freeze are not mutually exclusive. It is not ideal to build our logic in a way that goes agains the domain

Another challenge I see here, is that the data provenance between the AppMode and the Freeze is different. AppMode is a local setting for the user, while freeze is a session setting. 
I am unsure it makes sense for us to mix the concerns here, although I understand the convenience

I wonder if the correct place to consume this is from the runtime store. we could create a [feature selector](https://github.com/cpvalente/ontime/blob/master/apps/client/src/common/hooks/useSocket.ts) for it and consume it in the components instead of passing it as props. 

The other data is passed to the blocks as props, since we maintain the relationship of list > item, this is not the case here. Performance wise we would also prefer consuming this data as low down in the tree as possible

Reply:
Resolving with https://github.com/cpvalente/ontime/pull/1228/commits/e7ce68b39c1cd938a98b80b0a180d6d9acafe521